### ISSUE TITLE
Add sklearn.preprocessing.MinMaxScaler

### ISSUE THREAD

**Issue by wyddfrank** [mars-project/mars#1812]
Hi, I need to normalize my data before doing the real work, and I found sklearn.preprocessing.MinMaxScaler very useful. Could you help implement this module? Thanks a lot!
